examples/asr/asr_chunked_inference/aed/speech_to_text_aed_chunked_infer.py

```diff
@@ -33,13 +33,16 @@
 Example Usage:
 python speech_to_text_aed_chunked_infer.py \
     model_path=null \
-    pretrained_name="nvidia/canary-1b" \
+    pretrained_name="nvidia/canary-1b-flash" \
     audio_dir="<(optional) path to folder of audio files>" \
     dataset_manifest="<(optional) path to manifest>" \
     output_filename="<(optional) specify output filename>" \
     chunk_len_in_secs=40.0 \
     batch_size=16 \
-    decoding.beam.beam_size=5
+    decoding.beam.beam_size=1
+
+To return word and segment level timestamps, add `compute_timestamps=True` to the above command, 
```

Comment:
Can you update `compute_timestamps=True` to `timestamps=True` we updated this in transcribe_speech.py script as well.

Reply:
updated.